### PR TITLE
Homework 5: bus tickets

### PR DESCRIPTION
**Pull request by Nedzeya**

### CODE REVIEW

File: src/main/BusTicketReader.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.json.JSONTokener;
+
+public class BusTicketReader {
+    public static List<BusTicket> readTicketsFromFile(String filePath) throws IOException {
+        List<BusTicket> busTickets = new ArrayList<>();
+        BusTicketsValidator busTicketsValidator = new BusTicketsValidator();
+
+        try (BufferedReader reader = new BufferedReader(new FileReader(filePath))) {
```
Comment (Alexmansar) [inline src/main/BusTicketReader.java:25]: you can use catch as well

File: src/main/BusTicketsValidator.java [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+                && validateTicketType(busTicket);
+    }
+
+    /**
+     * only DAY,WEEK,YEAR types must have a startDate
+     */
+    private boolean validateHavingStartDate(BusTicket busTicket) {
+        TicketType ticketType = busTicket.getTicketType();
+        LocalDate startDate = busTicket.getStartDate();
+        if ((ticketType == TicketType.DAY ||
```
Comment (Alexmansar) [inline src/main/BusTicketsValidator.java]: use TickerType.Day.equals(ticketType) -> safer way (zeros)

**Review by Alexmansar — CHANGES_REQUESTED** [review]
@Nedzeya Hi, check please small comments)

File: src/main/BusTicketReader.java (hunk 1 above)
Comment (Nedzeya) [reply src/main/BusTicketReader.java:25]: done

**Review by Alexmansar — APPROVED** [review]
@Nedzeya Good job)) could be merged)

**Pull request merged**